### PR TITLE
Refreshed recipe and updated to v2.1.1

### PR DESCRIPTION
**Pull request by pelson**
Checklist:

* [X] Used a fork of the feedstock to propose changes
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by pelson** [comment]
@conda-forge-admin, please rerender

**Comment by pelson** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,20 +37,17 @@ test:
...
-    # https://github.com/SciTools/cf-units/pull/116 is merged and released.
-    # - cd cf_units && python -m pytest tests
+  commands:
+    - cd cf_units && python -m pytest tests
 
 about:
-  home: http://www.scitools.org.uk/iris
-  license: LGPL-3.0
+  home: https://scitools.org.uk/cf-units/docs/latest/
+  license: LGPLv3
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @pelson this is not a big deal and you don't have to revert it but we are trying to consolidate the licenses strings around the [SPDX](https://spdx.org/licenses/LGPL-3.0.html) standard.

**Comment by ocefpaf** [comment]
@pelson do you know if the Python 2.7 failures here are real or if we need to adjust the tests? (I'm not really following the changes upstream lately.)

**Comment by pelson** [comment]
They are kind of genuine. There are files in cf-units that are not python 2 compatible (the new TeX feature) - but I made sure I followed a well trodden path to ensure that those files never get imported in python 2 (else you get the SyntaxError we are seeing).

I'm actually going to improve the way the tests are done in this feedstock, as currently they are using the tests defined in the source repository on the package installed in site-packages, which is a bit weird IMO.

**Comment by pelson** [comment]
Looks like I've fixed what we were seeing, but now another (genuine?) test failure:

```
    def test_read_xml_invalid_path(self):
        with self.assertRaises(_ud.UdunitsError) as cm:
            _ud.read_xml(b'/not/a/path.xml')
        ex = cm.exception
    
        self.assertEqual(ex.status_msg(), 'UT_OPEN_ARG')
>       self.assertEqual(ex.errnum, errno.ENOENT)
E       AssertionError: 0 != 2
```

From what I can tell, this test was updated a few months ago in https://github.com/SciTools/cf-units/pull/116/files, and added a few years ago. For now, I suggest we skip this test on windows, and raise the issue on the cf-units tracker. Interestingly, this test is also *only* failing on Win and py27.

**Comment by ocefpaf** [comment]
Thanks Phil! I'm looking forward to play with the new latex functionality.

**Comment by pelson** [comment]
🎉 

> Thanks Phil! I'm looking forward to play with the new latex functionality.

Yay! I really like the idea of adding awesome functionality in python 3 only. 👍
Let me know if you come up with any issues with it (beyond the fact that we didn't implement Log units yet). Also, FWIW I've essentially implemented a UDUNITS2 parser - in theory we could consider submitting a change to the CF spec to talk about a units grammar, rather than mentioning a particular implementation (UDUNITS2) of said grammar. One for the future...

**Pull request merged**